pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerGauge.java

```diff
@@ -67,6 +67,12 @@
   // Percentage of segments we failed to get size for
   TABLE_STORAGE_EST_MISSING_SEGMENT_PERCENT("TableStorageEstMissingSegmentPercent", false),
 
+  // Number of errors during segment store upload retry of LLC segment
+  NUMBER_OF_ERRORS_FOR_LLC_SEGMENTS_DEEP_STORE_UPLOAD_RETRY("LLCSegmentDeepStoreUploadRetryError", false),
```

Comment:
Why gauge? These two should be meters.

Reply:
This change was based on your comment: https://github.com/apache/incubator-pinot/pull/6778#discussion_r644171605 .

Did I misunderstand this comment?

Reply:
These two metrics should be in Meter. The gauge needs to be the queue length --  total number of segments that are in your map.  I am sure a per-table gauge is also useful, just that it may cause number of metrics to explode. we can add that later if we see the need for it.

Reply:
Changed to Meter. I didn't add the Gauge about queue length though. The cache is in the format of Map.